TempoCore/Source/TempoTime/Private/TempoTimeServiceSubsystem.cpp

```diff
@@ -73,20 +80,29 @@ void UTempoTimeServiceSubsystem::SetSimStepsPerSecond(const SetSimStepsPerSecond
 		ResponseContinuation.ExecuteIfBound(TempoEmpty(), grpc::Status(grpc::StatusCode::OUT_OF_RANGE, "Steps per second must be >= 1"));
 		return;
 	}
-	
+
 	UTempoCoreSettings* Settings = GetMutableDefault<UTempoCoreSettings>();
 	Settings->SetSimulatedStepsPerSecond(RequestedStepsPerSecond);
 
+	if (!Cast<ATempoTimeWorldSettings>(GetWorld()->GetWorldSettings()))
+	{
+		ResponseContinuation.ExecuteIfBound(TempoEmpty(), grpc::Status(grpc::StatusCode::FAILED_PRECONDITION, "SetSimStepsPerSecond has no effect unless using TempoTimeWorldSettings"));
```

Comment:
Also, do this before setting the setting. The user would assume the setting has not been set when receiving and error.